packages/theme/src/themes/troms-theme/theme.ts

```diff
@@ -37,10 +37,11 @@ export const baseColors = {
   gray_1000: contrastColor('#000000', 'light'),
 
   // green
-  green_100: contrastColor('#F7FFF4', 'dark'),
-  green_200: contrastColor('#F3FFEE', 'dark'),
-  green_300: contrastColor('#E7FFDD', 'dark'),
-  green_400: contrastColor('#B2FF90', 'dark'),
+  green_50: contrastColor('#F7FFF4', 'dark'),
+  green_100: contrastColor('#F3FFEE', 'dark'),
+  green_200: contrastColor('#E7FFDD', 'dark'),
+  green_300: contrastColor('#B2FF90', 'dark'),
+  green_400: contrastColor('#a0e682', 'dark'),
```

Comment:
Excuse the nitpick 😄 

```suggestion
  green_400: contrastColor('#A0E682', 'dark'),
```

Reply:
Fixed now :)